### ISSUE TITLE
Imports work but are not imputed in the first cell on kaggle notebooks platform

### ISSUE THREAD

**Issue by miwojc** [8080labs/pyforest#50]
### Environment
* Platform: Kaggle Notebooks (https://www.kaggle.com/code)
* Operating System: Ubuntu 18.04.5 LTS
* Python Version: 3.7.10
* How did you install pyforest: ```!pip install upgrade pyforest``` (and restart notebook)
* Python packages: 
```
absl-py==0.12.0
affine==2.3.0
aiobotocore==1.3.1
aiohttp @ file:///home/conda/feedstock_root/build_artifacts/aiohttp_1623682242266/work
aiohttp-cors==0.7.0
aioitertools==0.7.1
aioredis==1.3.1
albumentations==1.0.1
alembic==1.6.5
allennlp==2.5.0
altair==4.1.0
annoy==1.17.0
ansiwrap==0.8.4
anyio @ file:///home/conda/feedstock_root/build_artifacts/anyio_1624037809245/work/dist
appdirs @ file:///home/conda/feedstock_root/build_artifacts/appdirs_1603108395799/work
argon2-cffi @ file:///home/conda/feedstock_root/build_artifacts/argon2-cffi_1610522577486/work
arrow @ file:///home/conda/feedstock_root/build_artifacts/arrow_1620325441783/work
arviz==0.11.2
asn1crypto @ file:///home/conda/feedstock_root/build_artifacts/asn1crypto_1595949944546/work
astunparse==1.6.3
async-generator==1.10
async-timeout==3.0.1
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1620387926260/work
audioread==2.1.9
autobahn==21.3.1
autocfg==0.0.8
Automat==20.2.0
Babel==2.9.1
backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work
backports.csv==1.0.7
backports.functools-lru-cache @ file:///home/conda/feedstock_root/build_artifacts/backports.functools_lru_cache_1618230623929/work
basemap @ file:///home/conda/feedstock_root/build_artifacts/basemap_1602682523538/work
bayesian-optimization==1.2.0
bayespy==0.5.22
beatrix-jupyterlab @ file:///tmp/beatrix_jupyterlab-latest.tar.gz
beautifulsoup4==4.9.3
binaryornot==0.4.4
biopython==1.79
black @ file:///home/conda/feedstock_root/build_artifacts/black-recipe_1622561163993/work
bleach @ file:///home/conda/feedstock_root/build_artifacts/bleach_1612213472466/work
blessings==1.7
blinker==1.4
blis==0.7.4
bokeh==2.3.2
Boruta==0.3
boto3==1.17.105
botocore==1.20.105
Bottleneck @ file:///home/conda/feedstock_root/build_artifacts/bottleneck_1611195599064/work
-e git+https://github.com/SohierDane/BigQuery_Helper@8615a7f6c1663e7f2d48aa2b32c2dbcb600a440f#egg=bq_helper
bqplot==0.12.29
branca==0.4.2
brewer2mpl==1.4.1
brotlipy==0.7.0
cachetools @ file:///home/conda/feedstock_root/build_artifacts/cachetools_1619593780156/work
caip-notebooks-serverextension @ file:///opt/conda/conda-bld/dlenv-base_1624064777121/work/packages/server_extension
Cartopy @ file:///home/conda/feedstock_root/build_artifacts/cartopy_1608577584581/work
catalogue==1.0.0
catalyst==21.6
catboost==0.26
category-encoders==2.2.2
certifi==2021.5.30
cesium==0.9.12
cffi @ file:///home/conda/feedstock_root/build_artifacts/cffi_1613413867554/work
cftime==1.5.0
chardet @ file:///home/conda/feedstock_root/build_artifacts/chardet_1610093487176/work
checklist==0.0.11
cheroot==8.5.2
CherryPy==18.6.1
cleverhans==3.1.0
click==7.1.2
click-plugins==1.1.1
cliff==3.8.0
cligj @ file:///home/conda/feedstock_root/build_artifacts/cligj_1622246659049/work
cloud-tpu-client==0.10
cloudpickle @ file:///home/conda/feedstock_root/build_artifacts/cloudpickle_1598400192773/work
cmaes==0.8.2
cmd2==2.1.2
cmdstanpy==0.9.5
cmudict==0.4.5
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1602866480661/work
colorcet==2.0.6
colorlog==5.0.1
colorlover==0.3.0
conda==4.9.2
conda-package-handling @ file:///home/conda/feedstock_root/build_artifacts/conda-package-handling_1618231393684/work
configparser==5.0.2
confuse @ file:///home/conda/feedstock_root/build_artifacts/confuse_1605988134315/work
constantly==15.1.0
contextily==1.1.0
contextlib2==21.6.0
convertdate==2.3.2
cookiecutter==1.7.0
cryptography @ file:///home/conda/feedstock_root/build_artifacts/cryptography_1616851478755/work
cufflinks==0.17.3
CVXcanon==0.1.2
cvxpy==1.1.7
cycler==0.10.0
cymem==2.0.5
cysignals==1.10.3
Cython==0.29.23
cytoolz==0.11.0
dask==2021.6.2
dataclasses==0.6
datashader==0.13.0
datashape==0.5.2
datatable==1.0.0
deap==1.3.1
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1621187651333/work
deepdish==0.3.6
defusedxml @ file:///home/conda/feedstock_root/build_artifacts/defusedxml_1615232257335/work
Delorean==1.0.0
Deprecated @ file:///home/conda/feedstock_root/build_artifacts/deprecated_1618254172504/work
descartes==1.1.0
dill==0.3.4
dipy==1.4.1
distributed==2021.6.2
dlib==19.22.0
dm-tree==0.1.6
docker @ file:///home/conda/feedstock_root/build_artifacts/docker-py_1617755742617/work
docker-pycreds==0.4.0
docutils==0.17.1
earthengine-api==0.1.272
easydev==0.11.1
easyocr==1.3.2
ecos==2.0.7.post1
eli5==0.11.0
emoji==1.2.0
en-core-web-lg @ https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-2.3.1/en_core_web_lg-2.3.1.tar.gz
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.3.1/en_core_web_sm-2.3.1.tar.gz
entrypoints @ file:///home/conda/feedstock_root/build_artifacts/entrypoints_1605121927639/work/dist/entrypoints-0.3-py2.py3-none-any.whl
ephem==4.0.0.2
essentia==2.1b6.dev374
fancyimpute==0.5.5
fastai==2.2.7
fastcore==1.3.20
fastprogress==1.0.0
fasttext==0.9.2
fbpca==1.0
fbprophet==0.7.1
feather-format==0.4.1
featuretools==0.25.0
feedparser==6.0.8
filelock==3.0.12
Fiona==1.8.18
fitter==1.3.0
flake8==3.9.2
flashtext==2.7
Flask==2.0.1
flatbuffers==1.12
folium==0.12.1
fsspec==2021.6.1
funcy==1.16
fury==0.7.0
future @ file:///home/conda/feedstock_root/build_artifacts/future_1610147336439/work
fuzzywuzzy==0.18.0
gast==0.3.3
gatspy==0.3
gcsfs @ file:///home/conda/feedstock_root/build_artifacts/gcsfs_1623161271836/work
GDAL==3.2.1
gensim==4.0.1
geographiclib==1.52
Geohash==1.0
geojson==2.5.0
geopandas @ file:///home/conda/feedstock_root/build_artifacts/geopandas_1623249625470/work
geoplot==0.4.3
geopy==2.1.0
geoviews==1.9.1
ggplot @ https://github.com/hbasria/ggpy/archive/0.11.5.zip
gitdb @ file:///home/conda/feedstock_root/build_artifacts/gitdb_1616781027402/work
GitPython @ file:///home/conda/feedstock_root/build_artifacts/gitpython_1623994295351/work
gluoncv==0.10.3
gluonnlp==0.10.0
google-api-core @ file:///home/conda/feedstock_root/build_artifacts/google-api-core-split_1623889342596/work
google-api-python-client==1.8.0
google-auth @ file:///home/conda/feedstock_root/build_artifacts/google-auth_1623897959686/work
google-auth-httplib2 @ file:///home/conda/feedstock_root/build_artifacts/google-auth-httplib2_1617387471894/work
google-auth-oauthlib @ file:///home/conda/feedstock_root/build_artifacts/google-auth-oauthlib_1618925959946/work
google-cloud-aiplatform @ git+https://github.com/googleapis/python-aiplatform.git@4ed7a50fef58d694ddb29d4240965d44e383da2b
google-cloud-appengine-logging==0.1.1
google-cloud-audit-log==0.1.0
google-cloud-automl==1.0.1
google-cloud-bigquery==2.2.0
google-cloud-bigtable==2.2.0
google-cloud-core==1.7.0
google-cloud-dataproc==2.4.0
google-cloud-datastore==2.1.3
google-cloud-firestore==2.1.3
google-cloud-kms==2.3.0
google-cloud-language==2.1.0
google-cloud-logging==2.5.0
google-cloud-monitoring==2.3.0
google-cloud-pubsub==1.7.0
google-cloud-scheduler==2.2.0
google-cloud-spanner==3.5.0
google-cloud-speech==2.4.0
google-cloud-storage==1.38.0
google-cloud-tasks==2.3.0
google-cloud-translate==3.2.0
google-cloud-videointelligence==2.2.0
google-cloud-vision==2.3.2
google-crc32c==1.1.2
google-pasta==0.2.0
google-resumable-media==1.3.1
googleapis-common-protos @ file:///home/conda/feedstock_root/build_artifacts/googleapis-common-protos-feedstock_1616710767141/work
gplearn==0.4.1
gpustat==0.6.0
gpxpy==1.4.2
graphviz==0.8.4
greenlet @ file:///home/conda/feedstock_root/build_artifacts/greenlet_1620327499468/work
grpc-google-iam-v1==0.12.3
grpcio==1.32.0
grpcio-gcp @ file:///home/conda/feedstock_root/build_artifacts/grpcio-gcp_1604414757985/work
gym==0.18.3
h2o==3.32.1.3
h5py==2.10.0
haversine==2.3.1
HeapDict==1.0.1
hep-ml==0.6.2
hijri-converter==2.1.3
hiredis==2.0.0
hmmlearn==0.2.5
holidays==0.11.1
holoviews==1.14.4
hpsklearn==0.1.0
html5lib==1.1
htmlmin==0.1.12
httplib2 @ file:///home/conda/feedstock_root/build_artifacts/httplib2_1617134439639/work
httplib2shim==0.0.3
huggingface-hub==0.0.8
humanize==3.10.0
hunspell==0.5.5
husl==4.0.3
hyperlink==21.0.0
hyperopt==0.2.5
hypertools==0.7.0
hypothesis==6.14.1
ibis-framework==1.4.0
idna @ file:///tmp/build/80754af9/idna_1593446292537/work
imagecodecs==2021.6.8
ImageHash @ file:///home/conda/feedstock_root/build_artifacts/imagehash_1605809872811/work
imageio==2.9.0
imbalanced-learn==0.8.0
imgaug==0.4.0
implicit==0.4.4
importlib-metadata==3.4.0
incremental==21.3.0
iniconfig==1.1.1
ipykernel @ file:///home/conda/feedstock_root/build_artifacts/ipykernel_1620912939357/work/dist/ipykernel-5.5.5-py3-none-any.whl
ipympl==0.7.0
ipython @ file:///home/conda/feedstock_root/build_artifacts/ipython_1622679237324/work
ipython-genutils==0.2.0
ipython-sql @ file:///home/conda/feedstock_root/build_artifacts/ipython-sql_1602667927030/work
ipywidgets @ file:///home/conda/feedstock_root/build_artifacts/ipywidgets_1609995587151/work
iso-639==0.4.5
iso3166==1.0.1
isoweek==1.3.3
itsdangerous==2.0.1
Janome==0.4.1
jaraco.classes==3.2.1
jaraco.collections==3.3.0
jaraco.functools==3.3.0
jaraco.text==3.5.0
jax==0.2.12
jaxlib==0.1.64
jedi @ file:///home/conda/feedstock_root/build_artifacts/jedi_1610146791823/work
jieba==0.42.1
Jinja2 @ file:///home/conda/feedstock_root/build_artifacts/jinja2_1621419064915/work
jinja2-time==0.2.0
jmespath==0.10.0
joblib @ file:///home/conda/feedstock_root/build_artifacts/joblib_1612898609989/work
json5 @ file:///home/conda/feedstock_root/build_artifacts/json5_1600692310011/work
jsonnet==0.17.0
jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema_1614815863336/work
jupyter==1.0.0
jupyter-client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1615693636836/work
jupyter-console==6.4.0
jupyter-core @ file:///home/conda/feedstock_root/build_artifacts/jupyter_core_1612125257337/work
jupyter-http-over-ws==0.0.8
jupyter-server @ file:///home/conda/feedstock_root/build_artifacts/jupyter_server_1621649383518/work
jupyter-server-mathjax @ file:///home/conda/feedstock_root/build_artifacts/jupyter-server-mathjax_1622646097887/work
jupyterlab==1.2.16
jupyterlab-git @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab-git_1598805204722/work
jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1601375948261/work
jupyterlab-server @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_server_1593951277307/work
jupyterlab-widgets @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_widgets_1609173350931/work
kaggle==1.5.12
kaggle-environments==1.7.11
Keras==2.4.3
Keras-Preprocessing==1.1.2
keras-tuner==1.0.3
kiwisolver @ file:///home/conda/feedstock_root/build_artifacts/kiwisolver_1610099773791/work
kmapper==2.0.1
kmodes==0.11.0
knnimpute==0.1.0
korean-lunar-calendar==0.2.1
kornia==0.5.5
kt-legacy==1.0.3
kubernetes @ file:///home/conda/feedstock_root/build_artifacts/python-kubernetes_1620952647333/work
langid==1.1.6
learntools @ git+https://github.com/Kaggle/learntools@68d31a9916e8ecbf6ff8637611ac7c20859672bd
leven==1.0.4
libcst==0.3.19
librosa==0.8.1
lightfm==1.16
lightgbm==3.2.1
lime==0.2.0.1
line-profiler==3.3.0
llvmlite==0.36.0
lmdb==1.2.1
lml==0.1.0
locket==0.2.1
LunarCalendar==0.0.9
lxml==4.6.3
Mako==1.1.4
mapclassify @ file:///home/conda/feedstock_root/build_artifacts/mapclassify_1608686039411/work
marisa-trie==0.7.5
Markdown @ file:///home/conda/feedstock_root/build_artifacts/markdown_1614595805172/work
markovify==0.9.0
MarkupSafe @ file:///home/conda/feedstock_root/build_artifacts/markupsafe_1621455667184/work
matplotlib==3.4.2
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1618935594181/work
matplotlib-venn==0.11.6
matrixprofile==1.1.10
mccabe==0.6.1
memory-profiler==0.58.0
mercantile==1.2.1
missingno==0.4.2
mistune @ file:///home/conda/feedstock_root/build_artifacts/mistune_1610112870915/work
mizani==0.7.3
ml-metrics==0.1.4
mlcrate==0.2.0
mlens==0.2.3
mlxtend==0.18.0
mmh3==3.0.0
mne==0.23.0
mnist==0.2.2
mock==4.0.3
more-itertools==8.8.0
mpld3==0.5.5
mpmath==1.2.1
msgpack==1.0.2
msgpack-numpy==0.4.7.1
multidict @ file:///home/conda/feedstock_root/build_artifacts/multidict_1610319002430/work
multimethod @ file:///home/conda/feedstock_root/build_artifacts/multimethod_1603129052241/work
multipledispatch==0.6.0
multiprocess==0.70.12.2
munch==2.5.0
murmurhash==1.0.5
mxnet==1.8.0.post0
mypy-extensions @ file:///home/conda/feedstock_root/build_artifacts/mypy_extensions_1610127163566/work
nb-conda @ file:///home/conda/feedstock_root/build_artifacts/nb_conda_1611345550379/work
nb-conda-kernels @ file:///home/conda/feedstock_root/build_artifacts/nb_conda_kernels_1606762466752/work
nbclient @ file:///home/conda/feedstock_root/build_artifacts/nbclient_1614336084111/work
nbconvert @ file:///home/conda/feedstock_root/build_artifacts/nbconvert_1605401831777/work
nbdime @ file:///home/conda/feedstock_root/build_artifacts/nbdime_1621970816386/work
nbformat @ file:///home/conda/feedstock_root/build_artifacts/nbformat_1617383142101/work
nest-asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1617163391303/work
netCDF4==1.5.7
networkx @ file:///home/conda/feedstock_root/build_artifacts/networkx_1598210780226/work
nibabel==3.2.1
nilearn==0.8.0
nltk==3.2.4
nnabla==1.20.1
nose==1.3.7
notebook @ file:///home/conda/feedstock_root/build_artifacts/notebook_1621259862661/work
notebook-executor @ file:///opt/conda/conda-bld/dlenv-base_1624064777121/work/packages/notebook_executor
numba @ file:///home/conda/feedstock_root/build_artifacts/numba_1623568548143/work
numexpr==2.7.3
numpy==1.19.5
nvidia-ml-py3==7.352.0
oauth2client==4.1.3
oauthlib @ file:///home/conda/feedstock_root/build_artifacts/oauthlib_1622563202229/work
odfpy==1.4.1
olefile @ file:///home/conda/feedstock_root/build_artifacts/olefile_1602866521163/work
onnx==1.8.1
opencensus==0.7.13
opencensus-context==0.1.2
opencv-python==4.5.2.54
opencv-python-headless==4.5.2.54
openslide-python==1.1.2
opt-einsum==3.3.0
optuna==2.8.0
orderedmultidict==1.0.1
ortools==9.0.9048
osmnx==1.1.1
osqp==0.6.2.post0
overrides==3.1.0
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1612459636436/work
palettable==3.3.0
pandarallel==1.5.2
pandas==1.2.4
pandas-datareader==0.9.0
pandas-profiling @ file:///home/conda/feedstock_root/build_artifacts/pandas-profiling_1620781969337/work
pandas-summary==0.0.7
pandasql==0.7.3
pandocfilters==1.4.2
panel==0.11.3
papermill @ file:///home/conda/feedstock_root/build_artifacts/papermill_1615479738199/work
param==1.11.1
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1617148930513/work
partd==1.2.0
path==16.0.0
path.py==12.5.0
pathos==0.2.8
pathspec @ file:///home/conda/feedstock_root/build_artifacts/pathspec_1605120834673/work
pathtools==0.1.2
patsy==0.5.1
patternfork-nosql==3.6
pbr==5.6.0
pdf2image==1.16.0
pdfminer.six==20201018
PDPbox @ git+https://github.com/SauceCat/PDPbox@b022a0aabcc6dbe2440244bf48d08fbb6ecdaf2d
pexpect @ file:///home/conda/feedstock_root/build_artifacts/pexpect_1602535608087/work
phik @ file:///home/conda/feedstock_root/build_artifacts/phik_1615724006677/work
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work
Pillow @ file:///home/conda/feedstock_root/build_artifacts/pillow_1621287993571/work
plac==1.1.3
plotly==5.1.0
plotly-express==0.4.1
plotnine==0.8.0
pluggy==0.13.1
polyglot==16.7.4
pooch==1.4.0
portalocker==2.3.0
portend==2.7.1
pox==0.3.0
poyo==0.5.0
ppca==0.0.4
ppft==1.6.6.4
preprocessing==0.1.13
preshed==3.0.5
prettytable @ file:///home/conda/feedstock_root/build_artifacts/prettytable_1614725168556/work
prometheus-client @ file:///home/conda/feedstock_root/build_artifacts/prometheus_client_1622586138406/work
promise==2.3
prompt-toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1623977816122/work
pronouncing==0.2.0
proto-plus==1.18.1
protobuf==3.17.3
psutil==5.8.0
ptyprocess @ file:///home/conda/feedstock_root/build_artifacts/ptyprocess_1609419310487/work/dist/ptyprocess-0.7.0-py2.py3-none-any.whl
pudb==2021.1
py==1.10.0
py-lz4framed==0.14.0
py-spy==0.3.7
py-stringmatching==0.4.2
py-stringsimjoin==0.3.2
pyaml==20.4.0
PyArabic==0.6.11
pyarrow==4.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.7
PyAstronomy==0.16.0
pybind11==2.6.2
pycodestyle==2.7.0
pycosat @ file:///home/conda/feedstock_root/build_artifacts/pycosat_1610094791171/work
pycountry==20.7.3
pycparser @ file:///tmp/build/80754af9/pycparser_1594388511720/work
pycrypto==2.6.1
pyct==0.4.8
pydantic @ file:///home/conda/feedstock_root/build_artifacts/pydantic_1620819958857/work
pydash==5.0.1
pydegensac==0.1.2
pyDeprecate==0.3.0
pydicom==2.1.2
pydot==1.4.2
pydub==0.25.1
pyemd==0.5.1
pyexcel-io==0.6.4
pyexcel-ods==0.6.0
pyfasttext==0.4.6
pyflakes==2.3.1
pyforest==1.1.0
pyglet==1.5.15
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1620245170812/work
PyJWT @ file:///home/conda/feedstock_root/build_artifacts/pyjwt_1619620644068/work
pykalman==0.9.5
pyLDAvis==3.3.1
pymc3==3.11.2
PyMeeus==0.5.11
pymongo==3.11.4
Pympler==0.9
pynndescent==0.5.3
pyocr==0.8
pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1608055815057/work
pyparsing==2.4.7
pyPdf==1.13
pyperclip==1.8.2
PyPrind==2.11.3
pyproj @ file:///home/conda/feedstock_root/build_artifacts/pyproj_1602664259410/work
PyQt5==5.12.3
PyQt5-sip==4.19.18
PyQtChart==5.12
PyQtWebEngine==5.12.1
pyrsistent @ file:///home/conda/feedstock_root/build_artifacts/pyrsistent_1610146801408/work
pysal==2.1.0
pyshp @ file:///home/conda/feedstock_root/build_artifacts/pyshp_1610646860606/work
PySocks @ file:///home/conda/feedstock_root/build_artifacts/pysocks_1610291444829/work
pystan==2.19.1.1
pytesseract==0.3.8
pytest==6.2.4
pytext-nlp==0.1.2
python-bidi==0.4.2
python-dateutil==2.8.1
python-docx==0.8.11
python-editor==1.0.4
python-igraph @ file:///home/conda/feedstock_root/build_artifacts/python-igraph_1625223698267/work
python-Levenshtein==0.12.2
python-louvain==0.15
python-slugify @ file:///home/conda/feedstock_root/build_artifacts/python-slugify_1620267765676/work
pytorch-ignite==0.4.5
pytorch-lightning==1.3.8
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1612179539967/work
pyu2f @ file:///home/conda/feedstock_root/build_artifacts/pyu2f_1604248910016/work
PyUpSet==0.1.1.post7
pyviz-comms==2.1.0
PyWavelets @ file:///home/conda/feedstock_root/build_artifacts/pywavelets_1607290802651/work
PyYAML==5.4.1
pyzmq @ file:///home/conda/feedstock_root/build_artifacts/pyzmq_1622038448462/work
qdldl==0.1.5.post0
qgrid==1.3.1
qtconsole==5.1.1
QtPy==1.9.0
randomgen==1.16.6
rasterio==1.2.6
ray==1.4.1
redis==3.5.3
regex @ file:///home/conda/feedstock_root/build_artifacts/regex_1617644420360/work
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1608156231189/work
requests-oauthlib @ file:///home/conda/feedstock_root/build_artifacts/requests-oauthlib_1595492159598/work
resampy==0.2.2
retrying==1.3.3
rgf-python==3.10.0
rsa @ file:///home/conda/feedstock_root/build_artifacts/rsa_1614171254180/work
Rtree @ file:///home/conda/feedstock_root/build_artifacts/rtree_1610147121172/work
ruamel-yaml-conda @ file:///home/conda/feedstock_root/build_artifacts/ruamel_yaml_1611943443937/work
s2sphere==0.2.5
s3fs==2021.6.1
s3transfer==0.4.2
sacremoses==0.0.45
scattertext==0.1.3
scikit-image==0.18.2
scikit-learn==0.23.2
scikit-multilearn==0.2.0
scikit-optimize==0.8.1
scikit-plot==0.3.7
scikit-surprise==1.1.1
scipy @ file:///home/conda/feedstock_root/build_artifacts/scipy_1619561915310/work
scs==2.1.4
seaborn @ file:///home/conda/feedstock_root/build_artifacts/seaborn-split_1611834504644/work
semver==2.13.0
Send2Trash==1.5.0
sentencepiece==0.1.96
sentry-sdk==1.2.0
setuptools-git==1.2
sgmllib3k==1.0.0
shap==0.39.0
Shapely @ file:///home/conda/feedstock_root/build_artifacts/shapely_1611317083224/work
shortuuid==1.0.1
SimpleITK==2.0.2
simplejson @ file:///home/conda/feedstock_root/build_artifacts/simplejson_1610147090454/work
six==1.15.0
sklearn==0.0
sklearn-contrib-py-earth @ git+git://github.com/scikit-learn-contrib/py-earth.git@dde5f899255411a7b9cbbabf93a817eff4b02e5e
sklearn-pandas==2.2.0
slicer==0.0.7
smart-open==5.1.0
smhasher==0.150.1
smmap @ file:///home/conda/feedstock_root/build_artifacts/smmap_1611376390914/work
sniffio @ file:///home/conda/feedstock_root/build_artifacts/sniffio_1610318316464/work
snuggs==1.4.7
sortedcontainers==2.4.0
SoundFile==0.10.3.post1
soupsieve==2.2.1
spacy==2.3.7
spectral==0.22.2
sphinx-rtd-theme==0.2.4
SQLAlchemy @ file:///home/conda/feedstock_root/build_artifacts/sqlalchemy_1623353958511/work
sqlparse @ file:///home/conda/feedstock_root/build_artifacts/sqlparse_1602142927465/work
squarify==0.4.3
srsly==1.0.5
statsmodels @ file:///home/conda/feedstock_root/build_artifacts/statsmodels_1612273600586/work
stemming==1.0.1
stevedore==3.3.0
stop-words==2018.7.23
stopit==1.1.2
stumpy==1.8.0
subprocess32==3.5.4
sympy==1.8
tables==3.6.1
tabulate==0.8.9
tangled-up-in-unicode @ file:///home/conda/feedstock_root/build_artifacts/tangled-up-in-unicode_1620720635379/work
tblib==1.7.0
tempora==4.1.1
tenacity @ file:///home/conda/feedstock_root/build_artifacts/tenacity_1614801055552/work
tensorboard==2.4.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
tensorboardX==2.2
tensorflow==2.4.1
tensorflow-addons==0.12.1
tensorflow-cloud==0.1.13
tensorflow-datasets==3.0.0
tensorflow-estimator==2.4.0
tensorflow-gcs-config==2.4.0
tensorflow-hub==0.12.0
tensorflow-metadata==1.1.0
tensorflow-probability==0.13.0
tensorpack==0.11
termcolor==1.1.0
terminado @ file:///home/conda/feedstock_root/build_artifacts/terminado_1623365324314/work
testpath @ file:///home/conda/feedstock_root/build_artifacts/testpath_1621261527237/work
text-unidecode==1.3
textblob==0.15.3
texttable @ file:///home/conda/feedstock_root/build_artifacts/texttable_1599404337529/work
textwrap3==0.9.2
Theano==1.0.5
Theano-PyMC==1.1.2
thinc==7.4.5
threadpoolctl @ file:///tmp/tmp79xdzxkt/threadpoolctl-2.1.0-py3-none-any.whl
tifffile==2021.7.2
tokenizers==0.10.3
toml @ file:///home/conda/feedstock_root/build_artifacts/toml_1604308577558/work
toolz==0.11.1
torch==1.7.0
torchaudio==0.7.0a0+ac17b64
torchmetrics==0.4.1
torchtext==0.8.0a0+cd6902d
torchvision==0.8.1
tornado @ file:///home/conda/feedstock_root/build_artifacts/tornado_1610094704591/work
TPOT==0.11.7
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1623518955413/work
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1602771532708/work
traittypes==0.2.1
transformers==4.6.1
trueskill==0.4.5
tsfresh==0.18.0
Twisted==21.2.0
txaio==21.2.1
typed-ast @ file:///home/conda/feedstock_root/build_artifacts/typed-ast_1618337592858/work
typeguard==2.12.1
typing-extensions==3.7.4.3
typing-inspect==0.7.1
tzlocal==2.1
ujson @ file:///home/conda/feedstock_root/build_artifacts/ujson_1611250187793/work
umap-learn==0.5.1
Unidecode @ file:///home/conda/feedstock_root/build_artifacts/unidecode_1612533191922/work
update-checker==0.18.0
uritemplate==3.0.1
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1622056799390/work
urwid==2.1.2
vecstack==0.4.0
visions @ file:///home/conda/feedstock_root/build_artifacts/visions_1612743227238/work
vowpalwabbit==8.10.2
vtk==9.0.3
Wand==0.6.6
wandb==0.10.33
wasabi==0.8.2
wavio==0.0.4
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1600965781394/work
webencodings==0.5.1
websocket-client @ file:///home/conda/feedstock_root/build_artifacts/websocket-client_1610127652206/work
Werkzeug==2.0.1
wfdb==3.4.0
whichcraft==0.6.1
widgetsnbextension @ file:///home/conda/feedstock_root/build_artifacts/widgetsnbextension_1605475530234/work
Wordbatch==1.4.8
wordcloud==1.8.1
wordsegment==1.3.1
wrapt @ file:///home/conda/feedstock_root/build_artifacts/wrapt_1610094880759/work
wslink==0.2.0
xarray==0.18.2
xgboost==1.4.2
xvfbwrapper==0.2.9
yacs==0.1.8
yarl @ file:///home/conda/feedstock_root/build_artifacts/yarl_1610354142409/work
yellowbrick==1.3.post1
zc.lockfile==2.0
zict==2.0.0
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1614945704755/work
zope.interface==5.4.0
```

### Description of Issue

* What did you expect to happen?
Imports work but are not inserted in the first cell
* What happened instead?
No visibility that something was imported

### Reproduction Steps

1. create new kaggle notebook
2. install pyforest with ```!pip install upgrade pyforest```
3. restart notebook
4. run ```np.sum([1, 2, 3])```

### What steps have you taken to resolve this already?
searched github issues, there is one closed by i still get the same issue
https://github.com/8080labs/pyforest/issues/31

### Anything else?
Thank you for your help!

![image](https://user-images.githubusercontent.com/32404415/129736169-02e87aef-5457-4f85-b216-d18cf1fcc3a1.png)

**Comment by miwojc** [comment]
Now, after reading the documentation, which i should have done in the first place i understand that pyforest is intended for use with Jupyter Notebook or Lab so closing this issue.

**Comment by FlorianWetschoreck** [comment]
Thank you for handling this issue yourself.
You are correct that the code export does not work in Kaggle notebooks (currently) because they do not allow/handle the notebook extension like Jupyter does.

As an alternative to the automatic code export, you can manually call `pyforest.active_imports()` to get the code that would be needed for the current imports.
